### ISSUE TITLE
Repeater fields not working in IE11/Edge

### ISSUE THREAD

**Issue by jwrudzin** [verbb/formie#316]
**Description**
Repeater fields are not working properly in IE11 or Edge. Clicking the button to add a row does not do anything.

**Steps to reproduce**
1. Add a repeater field to a form.
2. Using IE11 or Edge, click the button to add a row to the repeater field.
3. Nothing happens.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.13
- Craft version: 3.5.18
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.13",
  "...": "..."
}
```

Then run `composer update`.

Like https://github.com/verbb/formie/issues/317 and https://github.com/verbb/formie/issues/318 these work fine in Edge, but not in IE11 as reported. Can you confirm?

**Comment by engram-design** [comment]
Fixed in [1.3.14](https://github.com/verbb/formie/releases/tag/1.3.14)
